src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const rows = document.querySelectorAll('tr');
+
+for (const [index, row] of rows.entries()) {
+  if (index === 0 || index === rows.length - 1) {
+    const clonedCell = row.cells[4].cloneNode(true);
+
+    row.cells[4].textContent = row.cells[1].textContent;
+    row.appendChild(clonedCell);
+  } else {
+    const clonedCell = row.cells[4].cloneNode(true);
+
+    row.cells[4].textContent = row.cells[1].textContent;
+    row.appendChild(clonedCell);
+  }
```

Comment:
Now the code inside the if-else is duplicated, this is a bad practice, think about how you can optimize it to avoid code duplication